elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -160,12 +160,11 @@ protected CriterionFilterOperation buildCriterionFilterOperation(Criteria criter
             Optional<Pagination> pagination,
             RequestScope scope) {
 
-        if (pagination.isPresent()) {
-            pagination.get().evaluate(entityClass);
```

Comment:
Yes, it was moved to PersistentResource to keep evaluation of a pagination object outside of the datastores.